pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -814,3 +853,73 @@ def plot_channel_contributions_grid(
             ylabel="contribution",
         )
         return fig
+
+    def _validate_data(self, X, y=None):
+        return X
+
+    def sample_posterior_predictive(
+        self,
+        X_pred,
+        extend_idata,
```

Comment:
Default missing? Docstrings say it defaults to False, All other methods seem to have a default.

I am not sure what the default should be, probably True?

Reply:
I've changed all but the sample_prior_predictive (from model_builder) to default to True. Changed in the docstring as well